samples/pt-br/Primeiros Passos/index.html

```diff
@@ -14,46 +14,68 @@
     </p>
     
     
-    <h2>Nós estamos tentando algumas coisas novas</h2>
+    <!--
+        COMECE COM SEUS PRÓPRIOS ARQUIVOS
+    -->
+    
+    <h3>Projetos no Brackets</h3>
+    <p>
+        Para editar seu próprio código utilizando o Brackets, basta abrir a pasta contendo seus arquivos. Brackets trata a pasta atualmente aberta como um "Projeto", recursos como Code Hints, Live Preview e Quick Edit usam apenas os arquivos da pasta atualmente aberta.
+    </p>
+        
+    <samp>
+        Assim que você estiver pronto para sair desse projeto exemplo e editar seu próprio código, você pode usar o menu na barra lateral para trocar de pasta. Nesse momento, o menu diz "Primeiros Passos" - que é pasta que possui o arquivo que vocês está olhando agora mesmo. Clique no menu e escolhar "Abrir pasta..." para abrir sua própria pasta.
+        Você também pode usar o menu para voltar a pastas que você abriu anteriormente, incluindo esse projeto de exemplo.
+    </samp>
     
     <!--
         A RELAÇÃO ENTRE HTML, CSS E JAVASCRIPT
     -->
-    <h3>Edição Rápida de CSS e JavaScript</h3>
+        
+    <h3>Quick Edit (Edição Rápida) de CSS e JavaScript</h3>
     <p>
-        Ao editar HTML, use o atalho <kbd>Cmd/Ctrl + E</kbd> para abrir um editor rápido embutido que exibe todos os CSS relacionados. Faça um ajuste ao seu CSS, pressione <kbd>ESC</kbd> e você estará de volta editando HTML. Ou simplesmente deixe as regras CSS abertas e elas se tornarão parte de seu editor HTML.
-        Se você pressionar <kbd>ESC</kbd> fora de um editor rápido, todos eles vão recolher. Sem mais comutação entre documentos perdendo seu contexto.
+        Sem mais necessidade de ficar trocando entre seus arquivos e perder o contexto. Quando estiver editando HTML, use o atalho <kbd>Cmd/Ctrl + E</kbd> para abrir um rápido editor em linha (quick inline editor) que mostra todo o CSS relacionado.
+        Faça uma modificação no seu CSS, aperte <kbd>ESC</kbd> e você está de volta a edição do HTML, ou apenas deixe as regras CSS abertas e elas se tornarão parte do seu editor HTML. Se você apertar <kbd>ESC</kbd> fora do quick inline editor, todos eles irão se recolher. Quick Edit também encontrará regras definidas em arquivos LESS e SCSS, incluindo regras aninhadas.
     </p>
     
     <samp>
-        Quer vê-lo em ação? Coloque o cursor sobre o tag <!-- <samp> --> acima e pressione <kbd>Cmd/Ctrl + E</kbd>. Você deverá ver um editor rápido de CSS aparecer acima. À direita, você verá uma lista de regras CSS que estão relacionadas com esta tag.Simplesmente role as regras com <kbd>Alt + Up/Down</kbd> para encontrar o que deseja editar.
+        Quer vê-lo em ação? Coloque o cursor sobre o tag <!-- <samp> --> acima e pressione <kbd>Cmd/Ctrl + E</kbd>. Você deverá ver um editor rápido de CSS aparecer acima, mostrando as regras CSS que estão aplicadas a ele. Quick Edit funciona em classes e atribudos id também. Você pode usa-lo com seus arquivos LESS e SCSS também.
```

Comment:
* "Você pode usa-lo" -> "Você pode usá-lo "

I think "_a tag_" sounds better than "_o tag_". What you guys think?